lib/messages/push.js

```diff
@@ -35,7 +35,7 @@ class Push extends Message {
       attachments: [
         {
           ...super.getBaseMessage(),
-          fallback: this.summary,
+          fallback: `${this.summary} - ${this.push.repository.full_name}`,
```

Comment:
I think the old integration did `[owner/repo] - summary`. I don't have a strong preference either way. The only reason I could see to put repo name first is that it's typically a shorter and more predictable length than other parts of the message (maybe not specifically for push, but probably for other messages).